src/sushi/lib/constants.js

```diff
@@ -51,256 +60,130 @@ export const contractAddresses = {
 }
 
 /*
-UNI-V2 LP Address on mainnet for reference
+UNI-V2 LP Address on xdai for reference
 ==========================================
-0  USDT 0x0d4a11d5eeaac28ec3f61d100daf4d40471f1852
-1  USDC 0xb4e16d0168e52d35cacd2c6185b44281ec28c9dc
-2  DAI  0xa478c2975ab1ea89e8196811f51a7b7ade33eb11
-3  sUSD 0xf80758ab42c3b07da84053fd88804bcb6baa4b5c
-4  COMP 0xcffdded873554f362ac02f8fb1f02e5ada10516f
-5  LEND 0xab3f9bf1d81ddb224a2014e98b238638824bcf20
-6  SNX  0x43ae24960e5534731fc831386c07755a2dc33d47
-7  UMA  0x88d97d199b9ed37c29d846d00d443de980832a22
-8  LINK 0xa2107fa5b38d9bbd2c461d6edf11b11a50f6b974
-9  BAND 0xf421c3f2e695c2d4c0765379ccace8ade4a480d9
-10 AMPL 0xc5be99a02c6857f9eac67bbce58df5572498f40c
-11 YFI  0x2fdbadf3c4d5a8666bc06645b8358ab803996e28
-12 SUSHI 0xce84867c3c02b05dc570d0135103d3fb9cc19433
+HNY/WXDAI  0x4505b262dc053998c10685dc5f9098af8ae5c8ad
+HNY/STAKE  0x298c7326a6e4a6108b88520f285c7dc89403347d
+HNY/WETH   0x89e2f342b411032a580fefa17f96da6a5bef4112
+
 */
 
 export const supportedPools = [
-  {
-    pid: 12,
-    lpAddresses: {
-      1: '0xce84867c3c02b05dc570d0135103d3fb9cc19433',
-    },
-    tokenAddresses: {
-      1: '0x6b3595068778dd592e39a122f4f5a5cf09c90fe2',
-    },
-    name: 'Sushi Party!',
-    symbol: 'SUSHI-ETH UNI-V2 LP',
-    tokenSymbol: 'SUSHI',
-    icon: '🍣',
-  },
   {
     pid: 0,
     lpAddresses: {
-      1: '0x0d4a11d5eeaac28ec3f61d100daf4d40471f1852',
+      1: '0x4505b262dc053998c10685dc5f9098af8ae5c8ad',
     },
     tokenAddresses: {
-      1: '0xdac17f958d2ee523a2206206994597c13d831ec7',
+      1: '0xe91d153e0b41518a2ce8dd3d7944fa863463a97d',
     },
-    name: 'Tether Turtle',
-    symbol: 'USDT-ETH UNI-V2 LP',
-    tokenSymbol: 'USDT',
-    icon: '🐢',
+    name: 'XDAI Daisies',
+    symbol: 'XDAI-HNY UNI-V2 LP',
+    tokenSymbol: 'DAI',
+    icon: <img src={creamFlower} />,
```

Comment:
We probably want to give the image a CSS class to round it like the designs. I'm currently seeing it like:
<img width="1680" alt="Screen Shot 2020-09-22 at 8 32 24 PM" src="https://user-images.githubusercontent.com/9082013/93947683-bc620300-fd12-11ea-8456-96008fd5f6e2.png">